File: Software Technologies/Exam Preparation III - LogNoziroh & ShoppingList/LogNoziroh/JavaScript-Skeleton/models/Report.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    //TODO

    let Report = sequelize.define('Report', {
        status:{type:Sequelize.TEXT, required: true, allowNull: false,},
        message:{type:Sequelize.TEXT, required: true, allowNull: false,},
        origin:{type:Sequelize.TEXT, required: true, allowNull: false,},
    }, {
        timestamps:false
    })
    return Report;
};

File: Software Technologies/Exam Preparation III - LogNoziroh & ShoppingList/LogNoziroh/CSharp-Skeleton/LogNoziroh/Models/Report.cs
namespace LogNoziroh.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Report
    {
        //TODO
        [Key]
        public int Id { get; set; }

        public string Status { get; set; }

        public string Message { get; set; }

        public string Origin { get; set; }
    }
}
